HW3/11.py

```diff
@@ -0,0 +1,10 @@
+import random
+n = int(input())
+f = []
+for _ in range(n): f.append(random.randint(0,5))
```

Comment:
Такая запись хоть и допустима, но не рекомендуется
```suggestion
for _ in range(n):
    f.append(random.randint(0, 5))
```